### ISSUE TITLE
run downstream `pulumi-policy` tests

### ISSUE THREAD

**Issue by dixler** [pulumi/pulumi#11324]
During our postmortem call for #11140, we discussed running pulumi policy tests downstream to get more visibility into broken tests.

**Comment by Frassle** [comment]
I'd actually like us to consider deprecating the pulumi-policy repo and bringing it's code into this repo. My vision is that language SDK should support all the core parts of Pulumi. That is programs, providers, automation, and policy. I think that also ties in nicely with supporting things like inline providers and inline policy in programs.

**Comment by tyler-dunkel** [comment]
@Frassle did we ever move this?

**Comment by Frassle** [comment]
No, it's still on the list of things we think would be sensible to do. We do now have some tests in place to give confidence that if we moved everything into just the core library it still all works. But there's questions over what exactly we want to do. Do we keep these as separate npm/pypi libraries but just bring the source to pu/pu, or do we just merge everything into the main `@pulumi/pulumi` library, and if we do that how do we manage the deprecation of the pulumi-policy library.
